frontend/src/component/footer/Footer.jsx

```diff
@@ -42,7 +45,7 @@ const Footer = () => {
                                     </li>
                                     <li>
                                         <a href="https://twitter.com/ResourcioC" target="_blank" className="social-link">
-                                            <ion-icon name="logo-twitter"></ion-icon>
+                                            <img className='X-logo' src={x} />
                                         </a>
```

Comment:
Generally, we use 'Ion-icon' for the icons, but currently, the new logo of Twitter (X) is not available in Ion icon, so it's okay to use a PNG image for it